bgpd/bgp_attr.c

```diff
@@ -510,7 +510,14 @@ static bool bgp_attr_aigp_valid(uint8_t *pnt, int length)
 	}
 
 	while (length) {
+		uint8_t *end = data + length;
```

Comment:
doesn't end never change?  Why wouldn't we want to calcualte this before the while loop?

Reply:
Will move this outside the loop.

Reply:
Fixed, thanks.